test/compilable/betterCarray.d

```diff
@@ -4,7 +4,8 @@
 
 import core.stdc.stdio;
 
-extern (C) int main(char** argv, int argc) {
```

Comment:
Why remove these parameters?

Reply:
Because the signature was wrong and the parameters aren't used in the function body.

Reply:
I saw the latter, but not the former.